core_backend/app/question_answer/schemas.py

```diff
@@ -9,18 +9,28 @@
 
 class QueryBase(BaseModel):
     """
-    Pydantic model for query APIs
+    Question answering query base class.
     """
 
     query_text: str
     query_metadata: dict = {}
 
-    model_config = ConfigDict(from_attributes=True)
+    model_config = ConfigDict(
+        from_attributes=True,
+        json_schema_extra={
+            "examples": [
+                {
+                    "query_text": "What is AAQ?",
+                    "query_metadata": {"source": "optinal-metadata-example"},
```

Comment:
these also show up flipped... we really want the query text to come up first, could you look into it?